benchmarks/dynamo/common.py

```diff
@@ -86,7 +86,6 @@ class CI(NamedTuple):
     "tacotron2",
     # torchrec_dlrm requires gcc-11, https://github.com/pytorch/benchmark/pull/1427
     "torchrec_dlrm",
-    "nanogpt_generate",  # invalid multinomial distribution (sum of probabilities <= 0)
```

Comment:
Is it okay to also make this change. Should I kick off a run before tyring to land this? @voznesenskym @ngimel

Reply:
Looks like it isnt' erroring but is timing out. I created; https://github.com/pytorch/benchmark/pull/1659. To reduce the number of gen steps